### DISCUSSION TITLE
Is Strapi offline or unstable today?

### DISCUSSION THREAD

**Discussion by pedrogabrielLima** [strapi/strapi#7171] [Legacy Topics]
I tried to access Strapi.io today, and it's with a huge delay to open the site. I checked my internet and it's running ok, with a good speed and a good ping. I tested other sites and did a ping test to other organizations and everything is normal, except Strapi.io

**Comment by lauriejim** [comment]
It looks we have some trouble, will fix that soon as possible.
The documentation still available online. https://strapi.io/documentation
Or you can access it on GitHub.
